### DISCUSSION TITLE
Pytorch versions and memory consumption

### DISCUSSION THREAD

**Discussion by a-froghyar** [coqui-ai/TTS#985] [Qs for Live Q&A] [answered]
I had to dramatically decrease my batch size with the new pytorch/Coqui version, compared to March 2020. It'd be great if we could share a discussion about this

**Comment by skol101** [comment]
You should share your env details as well as what model/config you're training so we would be able to compare our results.

**Comment by erogol** [comment, marked as answer]
(answered in the call)
